### PR TITLE
Combat Bears

### PR DESCRIPTION
**Pull request by MMMiracles**
:cl: MMMiracles
add: A set of combat gear for bears has been added, see your local russian for more information.

/:cl:

Adds a set of combat gear that can be equipped to bears for various combat purposes. A combat kit can be salvaged from a soviet war bear, usually seen escorted around by drunk russians.

war1
war2
war3

Used darker outlining with a bit extra shaping so it looks a bit more like its shaped around a bear.
- [x] combat kit sprites
- [x] better armor sprites i dunno maybe

I may or may of not played an excess amount of C&C: RA as of recently.

### CODE REVIEW

**Comment by Lexorion** [comment]
Needs a shoulder mounted missile launcher, I presume this is admin spawn only?

**Comment by Niknakflak** [comment]
bad sprite

**Comment by MMMiracles** [comment]
@Lexorion glorious soviet war bear does not need shoulder-mounted missile launcher

@Niknakflak 'bad sprite' is no good criticism. details, da? is as close to reference material as one man can get

**Comment by FantasticFwoosh** [comment]
Bear cannot be ridden into battle by glorious soviet comrades, Premier Romanov is dissapointed. But commends your effort all the same

**Comment by Niknakflak** [comment]
It just looks kinda plain and not really like armor to me.  Da?

**Comment by optimumtact** [comment]
Just another boring bloat feature with no reason for existing :-1:

**Comment by MMMiracles** [comment]
ill just make bear armor dlc then.

**Comment by optimumtact** [comment]
I'll make your playing experience  A DLC

File: code/modules/mob/living/simple_animal/hostile/bear.dm [hunk 1]
```diff
@@ -54,12 +55,45 @@
...
 	health = 120
 	maxHealth = 120
 
 /mob/living/simple_animal/hostile/bear/Process_Spacemove(movement_dir = 0)
 	return 1	//No drifting in space for space bears!
 
+/mob/living/simple_animal/hostile/bear/update_icons()
+	..()
+	if(armored)
+		var/image/B = "armor_bear"
```
Comment (RemieRichards) [inline code/modules/mob/living/simple_animal/hostile/bear.dm]: uhhhhhhhhhhhhhhhhhhhhhhhhhhhhh

Comment (MMMiracles) [inline code/modules/mob/living/simple_animal/hostile/bear.dm]: it worked in-game please explain 'uhhhhhhhhhhhhhhhhhhhhhhhhhhhhh'

Comment (RemieRichards) [inline code/modules/mob/living/simple_animal/hostile/bear.dm]: it works by coincidence, this is not how you do images!
(the ```overlays``` list can take a string (```"armor_bear"```) to make an overlay from if there is a magic state in their icon file with that name, this "image/B" gets passed through ```add_overlay()``` and eventually ends up in the bear's ```overlays```, however B is typed as an image, when it is a string.)

make this ```var/image/B = image(icon = 'fucking_bear_thingy.dmi', icon_state = "armor_bear")```

Comment (MMMiracles) [inline code/modules/mob/living/simple_animal/hostile/bear.dm]: ~the more you know~

**Comment by TechnoAlchemisto** [comment]
the armor looks more like moss than armor, maybe try darker outlines and details?

**Comment by Razharas** [comment]
>There are still no ridable bears in game

How do you even play people i dont know

**Comment by ExcessiveUseOfCobblestone** [comment]
that bear is t h i c c

**Pull request merged**